### ISSUE TITLE
Pymodbus 3.7.4 broke Festo-Edcon again

### ISSUE THREAD

**Issue by roechele** [Festo-se/festo-edcon#7]
Pymodbus seemed to have chaged behaviour of "execute" Method which you are using in com_modbus.py.

It now requires a new Argument.

Pymodbus 3.7.3 execute method:
```
:param request: The request to process
```

Pymodbus 3.7.4 execute method Params:
```
:param no_response_expected: The client will not expect a response to the request
:param request: The request to process
```

Resolve it by sticking either to pymodbus v3.7.3 or Change code in com_modbus.py:

```
    def read_device_info(self) -> dict:
        """Reads device info from the CMMT and returns dict with containing values

        Returns:
            dict: Contains device information values
        """
        dev_info = {}

        # Read device information
        rreq = ReadDeviceInformationRequest(0x1, 0)
        print(rreq)
        rres = self.modbus_client.execute(False, rreq)
        dev_info["vendor_name"] = rres.information[0].decode("ascii")
        dev_info["product_code"] = rres.information[1].decode("ascii")
        dev_info["revision"] = rres.information[2].decode("ascii")

        rreq = ReadDeviceInformationRequest(0x2, 0)
        rres = self.modbus_client.execute(False, rreq)
        dev_info["vendor_url"] = rres.information[3].decode("ascii")
        dev_info["product_name"] = rres.information[4].decode("ascii")
        dev_info["model_name"] = rres.information[5].decode("ascii")

        for key, value in dev_info.items():
            Logging.logger.info(f"{key.replace('_',' ').title()}: {value}")

        return dev_info
```

Breaking chages in minor versions of pymodbus, not the first time:
https://github.com/Festo-se/festo-edcon/issues/6

**Comment by elrosch** [comment]
Oh dear.... Thanks for reporting.
I'll fix it and then gonna bind to that version since breaking changes seem to be a common pattern...

**Comment by elrosch** [comment]
Fixed in 0.14.10
